test/lit/passes/code-folding-eh-legacy.wast

```diff
@@ -352,9 +352,14 @@
   ;; CHECK-NEXT:    )
   ;; CHECK-NEXT:    (block
   ;; CHECK-NEXT:     (block
-  ;; CHECK-NEXT:      (drop
+  ;; CHECK-NEXT:      (if
   ;; CHECK-NEXT:       (local.get $0)
+  ;; CHECK-NEXT:       (then
+  ;; CHECK-NEXT:       )
+  ;; CHECK-NEXT:       (else
+  ;; CHECK-NEXT:       )
```

Comment:
This output pattern of empty if-elses was better optimized before, I think - we dropped the condition and left a single copy of the body (which in this case would have been a single nop).

Reply:
I see, got it. That makes sense. Maybe add that to the description/commit.

Though I think a few lines that turn `(if X (then) (else)` into `(drop X)` might be worth it for clarity and efficiency. That is, right after we optimize an if, we can see if the arms are empty and handle that. What do you think?

Reply:
Sure, that sounds good.